packages/ui/src/mocks/helpers/asChainData.ts

```diff
@@ -21,3 +22,8 @@ export const asChainData = (data: any): any => {
 }
 
 const withUnwrap = (data: Record<any, any>) => Object.defineProperty(data, 'unwrap', { value: () => data })
+// const withComputed = (data: Record<any, any>) => Object.defineProperties(data, {
+//   unwrap: { value: () => data },
+//   isSome: { value: Object.keys(data).length > 0 },
+//   keys: { value: () => of(Object.keys(data).map(entry => ({ args: [entry] }))) },
+// })
```

Comment:
Please remove the commented code on this file.

You're right the value can't be returned from the `asChainData`, instead something similar could be done on `asApiMethod` but it's not urgent right now...